front50-web/src/main/groovy/com/netflix/spinnaker/front50/controllers/DeliveryController.java

```diff
@@ -0,0 +1,77 @@
+package com.netflix.spinnaker.front50.controllers;
+
+import com.netflix.spinnaker.front50.exceptions.InvalidRequestException;
+import com.netflix.spinnaker.front50.model.delivery.Delivery;
+import com.netflix.spinnaker.front50.model.delivery.DeliveryRepository;
+import groovy.util.logging.Slf4j;
+import io.swagger.annotations.ApiOperation;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnExpression;
+import org.springframework.security.access.prepost.PostAuthorize;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Collection;
+
+@Slf4j
+@RestController
+@RequestMapping("")
+@ConditionalOnExpression("${spinnaker.delivery.enabled:false}")
+public class DeliveryController {
+
+  @Autowired
+  DeliveryRepository deliveryRepository;
```

Comment:
I'd suggest constructor injection as the better practice for new files (and a `private final DeliveryRepository deliveryRepository`).